dust_extinction/tests/test_warnings.py

```diff
@@ -14,6 +14,7 @@
     ave_models,
     grain_models,
 )
+from ..warnings import SpectralUnitsWarning
```

Comment:
I have been using absolute paths instead of relative throughout this package.  So, please change this to use absolute imports instead of relative for consistency.

Reply:
I retract this comment.  Both have been used in this repository.